workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -61,6 +61,18 @@ public bool Connect(ConnectionConfig config)
             return true;
         }
 
+        public void Disconnect()
+        {
+            if (Connection == null)
+            {
+                return;
```

Comment:
This probably shouldn't do nothing. Until we start looking at the broader error stuff I'd log this. I don't think this should do nothing though.